recipe/meta.yaml

```
@@ -1,15 +1,15 @@
-{% set version = "1.3" %}
+{% set version = "1.3.1" %}
 {% set p = "m2-" if win else "" %}
 
 package:
   name: libopus
   version: {{ version }}
 
 source:
-  url: https://archive.mozilla.org/pub/opus/opus-{{ version }}.1.tar.gz
+  url: https://archive.mozilla.org/pub/opus/opus-{{ version }}.tar.gz
   sha256: 65b58e1e25b2a114157014736a3d9dfeaad8d41be1c8179866f144a2fb44ff9d
 build:
-  number: 2
+  number: 1
```

```suggestion
  number: 0
```
The default build number is 0, not 1.

---

On some platforms it already exists with build number `0`:
```
conda search libopus --platform=osx-64
Loading channels: done
# Name                       Version           Build  Channel
libopus                        1.2.1      h169cedb_0  pkgs/main
libopus                          1.3      h1de35cc_0  pkgs/main
libopus                          1.3      h46256e1_2  pkgs/main
libopus                        1.3.1      h1de35cc_0  pkgs/main
```

---

I considered the same, added a comment in the description, the log of conda search was already there

---

Notice also that two versions have the same hash name
```
libopus                          1.3      h7b6447c_0  main
libopus                        1.3.1      h7b6447c_0  main
```

---

@lorepirri two versions having the same package hash is fine in this case because they are different versions.

---

ah ok, right